pipelines/batch/src/main/java/org/openmrs/analytics/FhirEtlOptions.java

```diff
@@ -133,6 +133,12 @@ public interface FhirEtlOptions extends PipelineOptions {
 	
 	void setJdbcModeHapi(Boolean value);
 	
+	@Description("The batch size for jdbc mode batch extract for a HAPI source")
+	@Default.Integer(100000)
+	Integer getHapiJdbcBatchSize();
```

Comment:
So as discussed offline, let's consolidate this and the other batch size option above `getBatchSize` and update its description. I think we are adding too many options which can be confusing.